bot/handlers/administration/admin_help_handler.py

```diff
@@ -15,8 +15,12 @@ class AdminHelpHandler(BotMessageHandler):
     def get_commands(self) -> List[str]:
         return ["admin"]
 
+    async def is_any_validation_failed(self, message: Message) -> bool:
+        return False
```

Comment:
możesz też zrobić w klasie bazowej funkcję wirtualną która ma zwracać listę funkcji do wykonania

wtedy is_any_validation_failed ma JEDNĄ implementację w klasie bazowej

```py
for validator in self.__get_validator_functions():
    if not validator(): return False
return True
```

Tylko musiałbyś pokminić nad wspólnym interfejsem (args) funkcji validator, ale jak to zrobisz to będzie **najczystsze** rozwiązanie z możliwych, jeszcze lepsze niż to co pisałem wcześniej